gui-libs/aquamarine/aquamarine-0.1.1-r2.ebuild

```diff
@@ -19,6 +19,8 @@ fi
 LICENSE="BSD"
 SLOT="0"
 
+# Upstream states that the simpleWindow test is broken.
```

Comment:
To better track this in the future  for other maintainers
I'd suggest also append 'bug 936653' to the end of comment, also any URL link from upstream?